api/RTCPeerConnection.json

```diff
@@ -2778,195 +2628,6 @@
           }
         }
       },
-      "onidentityresult": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/onidentityresult",
-          "support": {
-            "chrome": {
-              "version_added": false
-            },
-            "chrome_android": {
-              "version_added": false
-            },
-            "edge": {
-              "version_added": "≤18",
-              "version_removed": "79"
-            },
-            "firefox": {
-              "version_added": "22"
-            },
-            "firefox_android": {
-              "version_added": "44"
-            },
-            "ie": {
-              "version_added": false
-            },
-            "opera": [
```

Comment:
This data doesn't make any sense, but to be sure I've disconfirmed it in Opera 77 using https://mdn-bcd-collector.appspot.com/tests/api/RTCPeerConnection/onidentityresult.